doc/rbd/iscsi-target-ansible.rst

```diff
@@ -0,0 +1,360 @@
+==========================================
+Configuring the iSCSI Target using Ansible
+==========================================
+
+The Ceph iSCSI gateway is the iSCSI target node and also a Ceph client
+node. The Ceph iSCSI gateway can be a standalone node or be colocated on
+a Ceph Object Store Disk (OSD) node. Completing the following steps will
+install, and configure the Ceph iSCSI gateway for basic operation.
+
+**Requirements:**
+
+-  A running Ceph Jewel (10.2.x) cluster or newer
+
+-  The ``device-mapper-multipath-0.4.9-99`` or newer package
+
+-  The ``targetcli-2.1.fb41-3`` or newer package
```

Comment:
I don't think ceph-iscsi-ansible directly requires ```targetcli``` -- instead, maybe list ```ceph-iscsi-config``` and related packages (from the requirements page) needing to be available on the gateway nodes?